src/charm.py

```diff
@@ -163,8 +163,8 @@ def _remove_unlisted_plugins(self, container: ops.Container) -> ops.StatusBase:
             logger.error("Failed to remove unlisted plugin, %s", exc)
             return ops.StatusBase.from_name(original_status, "Failed to remove unlisted plugin.")
         except TimeoutError as exc:
-            logger.error("Failed to restart jenkins after removing plugin, %s", exc)
-            return ops.BlockedStatus("Failed to restart Jenkins after removing plugins")
+            logger.error("Failed to remove plugins, %s", exc)
+            return ops.BlockedStatus("Failed to remove plugins.")
```

Comment:
Since is a blocked status, what would be the action that the user can take to solve this?

Reply:
I think there was a discussion around `DegradedStatus` but it isn't quite supported yet. This is to indicate that the service may be down or degraded.
In order to resolve it, the user must either:
1. Restart Jenkins via pebble again if Jenkins has failed to restart for some reason
2. Resolve the plugin remove failure (which is usually due to dependency problems)

Reply:
Nice, do you think it could be described in the message or log?

Reply:
WIll create a followup PR since it's break right now :D